tests/test_models.py

```diff
@@ -678,7 +668,8 @@ def test_creation_time(self, db_session):
 
     def test_create_participant(self, db_session):
         participant = models.Participant(
-            worker_id=str(1), hit_id=str(1), assignment_id=str(1), mode="test")
+            recruiter_id='hotair', worker_id=str(1), hit_id=str(1),
```

Comment:
Since we're touching the code, any reason not to move to the `a.participant()` factory in these tests?